modules/tile-converter/src/i3s-converter/helpers/geometry-attributes.ts

```diff
@@ -79,6 +80,19 @@ function calculateFaceRangesAndFeaturesCount(featureIndices: number[]): {
   return {faceRange, featureCount, featureIds};
 }
 
+/**
+ * Find most frequent value to avoid situation where one vertex can be part of multiple features (objects).
+ * @param values
+ */
+function getFrequentValue(values: number[]): number {
+  const map: {[key: number]: number} = {};
+  for (const value of values) {
+    map[value] = (map[value] || 0) + 1;
+  }
+  const sorted: [string, number][] = Object.entries(map).sort((a, b) => b[1] - a[1]);
```

Comment:
Just use `Object.values()`?

Reply:
Changed this method completely. Now I find the most frequent value in one pass through the array.